lib/src/status.rs

```diff
@@ -324,40 +325,62 @@ pub(crate) async fn status(opts: super::cli::StatusOpts) -> Result<()> {
     Ok(())
 }
 
+/// Write the data for a container image based status.
+fn human_render_imagestatus(
+    mut out: impl Write,
+    slot_name: &str,
+    image: &crate::spec::ImageStatus,
+) -> Result<()> {
+    let transport = &image.image.transport;
+    let imagename = &image.image.image;
+    // Registry is the default, so don't show that
+    let imageref = if transport == "registry" {
+        Cow::Borrowed(imagename)
+    } else {
+        // But for non-registry we include the transport
+        Cow::Owned(format!("{transport}:{imagename}"))
+    };
+    writeln!(out, "Current {slot_name} image: {imageref}")?;
+
+    let version = image
+        .version
+        .as_deref()
+        .unwrap_or("No image version defined");
+    let timestamp = image
+        .timestamp
+        .as_ref()
+        .map(|t| t.to_string())
+        .unwrap_or_else(|| "No timestamp present".to_owned());
+    let digest = &image.image_digest;
+
+    writeln!(out, "    Image version: {version} ({timestamp})")?;
+    writeln!(out, "    Image digest: {digest}")?;
+    Ok(())
+}
+
+fn human_render_ostree(mut out: impl Write, slot_name: &str, _ostree_commit: &str) -> Result<()> {
```

Comment:
Is this where we'd implement the documentation for additional deployments like `rpm-ostree status`?

Reply:
Yep